orion-server/src/main/java/com/pinterest/orion/core/actions/aws/AmiTagManager.java

```diff
@@ -0,0 +1,100 @@
+package com.pinterest.orion.core.actions.aws;
```

Comment:
Please add copyright block in all new files.

Reply:
Done